app/src/main/java/com/framgia/fsalon/screen/scheduler/detail/BookingDetailViewModel.java

```diff
@@ -231,4 +341,96 @@ public void setChangeStatus(boolean changeStatus) {
         mIsChangeStatus = changeStatus;
         notifyPropertyChanged(BR.changeStatus);
     }
+
+    public AppCompatActivity getActivity() {
+        return mActivity;
+    }
+
+    public void setActivity(AppCompatActivity activity) {
+        mActivity = activity;
+        notifyPropertyChanged(BR.activity);
+    }
+
+    @Bindable
+    public String getPathImageSideLeft() {
+        return mPathImageSideLeft;
+    }
+
+    public void setPathImageSideLeft(String pathImageSideLeft) {
+        mPathImageSideLeft = pathImageSideLeft;
+        notifyPropertyChanged(BR.pathImageSideLeft);
+    }
+
+    @Bindable
+    public String getPathImageSideRight() {
+        return mPathImageSideRight;
+    }
+
+    public void setPathImageSideRight(String pathImageSideRight) {
+        mPathImageSideRight = pathImageSideRight;
+        notifyPropertyChanged(BR.pathImageSideRight);
+    }
+
+    @Bindable
+    public String getPathImageSideBehind() {
+        return mPathImageSideBehind;
+    }
+
+    public void setPathImageSideBehind(String pathImageSideBehind) {
+        mPathImageSideBehind = pathImageSideBehind;
+        notifyPropertyChanged(BR.pathImageSideBehind);
+    }
+
+    @Bindable
+    public String getPathImageSideFrontUp() {
+        return mPathImageSideFrontUp;
+    }
+
+    public void setPathImageSideFrontUp(String pathImageSideFrontUp) {
+        mPathImageSideFrontUp = pathImageSideFrontUp;
+        notifyPropertyChanged(BR.pathImageSideFrontUp);
+    }
+
+    public int getSidePhoto() {
+        return mSidePhoto;
+    }
+
+    public void setSidePhoto(int sidePhoto) {
+        mSidePhoto = sidePhoto;
+    }
+
+    private Uri getPickImageResultUri(Intent data) {
+        boolean isCamera = false;
+        if (data != null) {
+            Bundle bundle = data.getExtras();
+            if (bundle != null) {
+                isCamera = bundle.getBoolean(BUNDLE_IS_CAPTURED);
+            }
+        }
+        return isCamera ? getCaptureImageOutputUri() : data.getData();
+    }
+
+    /**
+     * Get URI to image received from capture by camera.
+     */
+    private Uri getCaptureImageOutputUri() {
+        Uri outputFileUri = null;
+        File imageFile = mActivity.getExternalCacheDir();
+        if (imageFile != null) {
+            outputFileUri = Uri.fromFile(new File(imageFile.getPath(), TEMP_IMAGE_NAME));
+        }
+        return outputFileUri;
+    }
+
+    /**
+     * Define all side of customer 's photos
+     */
+    @IntDef({SIDE_LEFT, SIDE_RIGHT, SIDE_BEHIND, SIDE_FRONT_UP})
+    @Target(ElementType.PARAMETER)
+    public @interface SideCapture {
+        int SIDE_LEFT = 0;
```

Comment:
ERROR: 'member def type' have incorrect indentation level 8, expected level should be 4.